src/javascript/EditPanel/EditPanelContent/FormBuilder/Sections/Section/Section.jsx

```diff
@@ -5,15 +5,23 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import {SectionPropTypes} from '~/FormDefinitions';
 
-export const Section = ({section, isExpanded, onClick}) => (
-    <Collapsible data-sel-content-editor-fields-group={section.displayName}
-                 label={section.displayName}
-                 isExpanded={isExpanded}
-                 onClick={onClick}
-    >
-        <FieldSetsDisplay fieldSets={filterRegularFieldSets(section.fieldSets)}/>
-    </Collapsible>
-);
+export const Section = ({section, isExpanded, onClick}) => {
+    const fieldSets = filterRegularFieldSets(section.fieldSets);
+
+    if (fieldSets.length === 0) {
+        return null;
+    }
+
+    return (
+        <Collapsible data-sel-content-editor-fields-group={section.displayName}
+                     label={section.displayName}
+                     isExpanded={isExpanded}
+                     onClick={onClick}
+        >
+            <FieldSetsDisplay fieldSets={filterRegularFieldSets(section.fieldSets)}/>
```

Comment:
possible to use `fieldSets` variable here instead of running through filter twice?